ethereum-types/src/hash.rs

```diff
@@ -94,24 +94,24 @@ impl From<H256> for H160 {
 #[deprecated]
 impl From<H256> for H64 {
```

Comment:
Maybe we can remove these `deprecated` methods now?

Reply:
yup! good idea

Reply:
Please do note that I wanted the changes in this crate to be minimal to the outside ecosystem so that I can refocus on other work as soon as possible instead of fixing all the to-be-broken depending crates. That's the reason I did not change some of the things that should have been done otherwise such as renaming the crate features etc.